_posts/2020-07-29-#17977.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: pr
+date: 2020-07-29
+title: "Implement BIP 340-342 validation - Implement Taproot signature hashing"
+pr: 17977
+authors: [sipa]
+components: ["consensus"]
+host: jnewbery
+status: upcoming
+commit: ad51604f57aaa311529bd302e069c26dac818ad6
+---
+
+This is the third Review Club meeting on the (work in progress) implementation
+of BIP 340-342.  We've previously looked at:
+
+- [O(1) OP_IF/NOTIF/ELSE/ENDIF script implementation / Abstract out script
+  execution out of VerifyWitnessProgram()](./16902.html)
+- [Initialize PrecomputedTransactionData in CheckInputScripts and Move
+  single-sig checking EvalScript code to EvalChecksig](./18401.html)
+
+Those preliminary changes have been merged into the master branch in separate PRs.
+
+This week, we'll look at one commit from PR 17977 - _[Implement Taproot
+signature
+hashing](https://github.com/bitcoin-core-review-club/bitcoin/commit/41d08f5d77f52bec0e31bb081d85fff2d67d0467)_.
+
+**Participants are encouraged to review [the notes from the session on
+PR18401](./18401.html) in particular, since we'll be covering a lot of the same
```

Comment:
nit if you retouch: can omit `.html` here and in line 30

Reply:
done